rust/build.rs

```diff
@@ -5,6 +5,12 @@ mod platform_cfg {
         let ver = glibc_version::get_version().unwrap();
         if ver.major >= 2 && ver.minor >= 28 {
             println!("cargo:rustc-cfg=glibc_renameat2");
+        } else {
+            let more_info = "https://docs.rs/deltalake/0.4.0/deltalake/storage/file/struct.FileStorageBackend.html";
```

Comment:
would be better to point the doc url to latest version to pick up changes automateically.

```suggestion
            let more_info = "https://docs.rs/deltalake/latest/deltalake/storage/file/struct.FileStorageBackend.html";
```